server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -2857,7 +2857,7 @@ private DiskOfferingVO createDiskOfferingInternal(final long userId, final boole
                                                       Long iopsWriteRate, Long iopsWriteRateMax, Long iopsWriteRateMaxLength,
                                                       final Integer hypervisorSnapshotReserve, String cacheMode, final Long storagePolicyID) {
 
-        DiskOfferingVO diskOffering = new DiskOfferingVO(name, displayText, typedProvisioningType, false, tags, false, localStorageRequired, true);
+        DiskOfferingVO diskOffering = new DiskOfferingVO(name, displayText, typedProvisioningType, false, tags, false, localStorageRequired, false);
```

Comment:
Can you check if 'customized' flag has to be set for disk offering, when `rootDiskSizeInGiB == 0`? Or Is it set else where?

Reply:
This method createDiskOfferingInternal is used only to create disk offering entry for compute only service offering. In this case customized is always false since compute offering creation does not allow creating customized disk size disk offering.